### PR TITLE
Toms Prod And Dev additions

### PR DESCRIPTION
**Pull request by TBurnip**

### CODE REVIEW

File: wad_project/socshare/views.py [hunk 1]
```diff
@@ -62,7 +63,23 @@ def register(request):
...
+                    user.email = email
+                    user.set_password(password)
+                    user.save()
+                    society = Society.objects.get_or_create(name=name, user=user)[0]
+                    society.acronym = acronym
+                    society.save()
+                    login(request, user)
+                    return redirect(reverse('socshare:events'))
+                else:
+                    return render(request,'socshare/register.html',context={'alert':'warning','alert_msg':'Account not registered with SRC!'})
```
Comment (fork-bombed) [inline wad_project/socshare/views.py]: We shouldn't be reusing this much code. Find a more modular way to do it.

File: wad_project/wad_project/settings.py [hunk 2]
```diff
@@ -29,10 +29,13 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'h8m5o)8vv&vtm69$kcxo1-9i21%87=vb=$t%42cc^np-u^bb4@'
```
Comment (ProstoSanja) [inline wad_project/wad_project/settings.py]: emmm??????

File: wad_project/wad_project/settings.py [hunk 3]
```diff
@@ -29,10 +29,13 @@
...
 
+# This tell us if we are in production or not
+PROD = False
+
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = not PROD
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["socshare.pythonanywhere.com"] + ["localhost"] if not PROD else []
```
Comment (ProstoSanja) [inline wad_project/wad_project/settings.py]: isnt it other way round?

Comment (TBurnip) [reply wad_project/wad_project/settings.py]: no

Comment (TBurnip) [reply wad_project/wad_project/settings.py]: when the array isn't empty it will shout at you if you are not using an allowed host

**Comment by ProstoSanja** [comment]
are we actually keeping prod key in there??

**Pull request merged**